9.2/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM centos:centos7
+
+# PostgreSQL image for OpenShift.
+# Volumes:
+#  * /var/lib/psql/data   - Database cluster for PostgreSQL
+# Environment:
+#  * $POSTGRESQL_USER     - Database user name
+#  * $POSTGRESQL_PASSWORD - User's password
+#  * $POSTGRESQL_DATABASE - Name of the database to create
+#  * $POSTGRESQL_ADMIN_PASSWORD (Optional) - Password for the 'postgres'
+#                           PostgreSQL administrative account
+
+# Image metadata
+ENV POSTGRESQL_VERSION      9.2
+ENV IMAGE_DESCRIPTION       PostgreSQL 9.2
+ENV IMAGE_TAGS              postgresql,postgresql92
+ENV IMAGE_EXPOSE_SERVICES   5432:postgresql
+
+# These are relevant to PostgreSQL
+ENV PGDATA /var/lib/pgsql/data
+# TODO: We should consider if this should be changeable. Can affect performance.
+#       http://www.postgresql.org/docs/9.2/static/locale.html
+ENV LANG en_US.utf8
+
+MAINTAINER  Martin Nagy <mnagy@redhat.com>
+EXPOSE 5432
+
+# This image must forever use UID 26 for postgres user so our volumes are
+# safe in the future. This should *never* change, the last test is there
+# to make sure of that.
+RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
+    yum -y --setopt=tsflags=nodocs install https://www.softwarecollections.org/en/scls/rhscl/postgresql92/epel-7-x86_64/download/rhscl-postgresql92-epel-7-x86_64.noarch.rpm && \
+    yum -y --setopt=tsflags=nodocs install postgresql92 && \
+    yum clean all && \
+    test "$(id postgres)" = "uid=26(postgres) gid=26(postgres) groups=26(postgres)"
+
+COPY run-postgresql.sh /usr/local/bin/
+
+VOLUME ["/var/lib/pgsql/data"]
```

Comment:
/var/lib/pgsql/data is a commonly used path for a database cluster, which also contains within the configuration that is necessary, so config files are persistently stored. It didn't make much sense to me to use /var/lib/pgsql. Besides, I think that while experimenting I did see some psql history files there, or something like that. Probably not relevant to us, but stil..